README.md

```diff
@@ -1,77 +1,137 @@
+     - [Education](#-education)
+     - [Funding](#-funding)
+     - [Frameworks, Platforms and Libraries](#-frameworks-platforms-and-libraries)
+     - [Gaming](#-gaming)
+     - [Game Consoles](#%EF%B8%8F-game-consoles)
+     - [Hosting/SaaS](#-hostingsaas)
+     - [IDEs/Editors](#-ideseditors)
+     - [Languages](#-languages)
+     - [ML/DL](#-mldl)
+     - [Music](#-music)
+     - [Office](#-office)
+     - [Operating System](#%EF%B8%8F-operating-system)
+     - [ORM](#-orm)
+     - [Other](#-other)
+     - [Quantum Programming Frameworks and Libraries](#quantum-programming-frameworks-and-libraries)
+     - [Search Engines](#search-engines)
+     - [Servers](#servers)
+     - [Smartphone Brands](#smartphone-brands)
+     - [Social](#social)
+     - [Store](#store)
+     - [Streaming](#streaming)
+     - [Testing](#testing)
+     - [Version Control](#version-control)
+     - [Work/Jobs](#workjobs)
+   </details>
+
+
+# Usage
+To use a badge:
+- Via Github
+    1. Press `Ctrl` + `f` on your keyboard, to bring out the search modal.
+    1. Enter the name of the badge you need.
+    1. Copy the appropriate `![Name](link)` element and paste it in your Markdown file (e.g. README.md)
+- You could also visit the live site at [ileriayo.github.io/markdown-badges/](https://ileriayo.github.io/markdown-badges/)
+
+# Tips
+
+<details> 
+<summary>👇 How to use a different badge style</summary>
+<hr>
+
+> <strong>Note:</strong> `for-the-badge` is the style that we chose for appearance purposes. Other styles are available at [https://shields.io/#styles](https://shields.io/#styles) and can be used with the badges here. Thanks, @kingthorin for mentioning this!
+
+
+Shields.io offers 4 styles, which are:
+- Plastic 
+- For-the-badge
+- Social  
+- Flat-square 
+- Flat 
+
+💡 To use a different style: Replace `for-the-badge` in the markdown link with any of the styles above.
+```
+Example.
+
+🧷  For plastic
+https://shields.io/badge/style-plastic-green?logo=appveyor&style=plastic
+
+🤏🏽  For Flat
+https://shields.io/badge/style-flat-green?logo=appveyor&style=flat
+```
+
+</details>
+
+<details> 
+<summary>👇 Use Ctrl + F or CMD + F to search</summary>
+ <hr>
+
+ > <strong>Tip:</strong> Since there are a lot of badges, to search for the particular badge you are looking for, use Ctrl + F and type the name you want. Thanks, @JakyeRU for mentioning this!
+</details>
+
+
+## Contribution
+
+The project has a separate contribution file. Please adhere to the steps listed in the separate contributions [file](./CONTRIBUTING.md)
+
+## Contact
+
+You can reach me on [Twitter @ileriayooo](https://twitter.com/Ileriayooo)
+
+## License
+
+[![Licence](https://img.shields.io/github/license/Ileriayo/markdown-badges?style=for-the-badge)](./LICENSE)
+```
+////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////////////////////
+//////////////////////////////////////////////////////////////////
```

Comment:
Just checking.
Is this some typo? 
<img width="1067" alt="Screenshot 2022-03-19 at 09 35 34" src="https://user-images.githubusercontent.com/13166712/159114061-bc57cc40-4714-4a86-adad-0b99c215c898.png">

Reply:
😄 I needed a way to add breathing space between the badge section and the license.

Reply:
😄